app/browser/api/ledgerNotifications.js

```diff
@@ -42,8 +42,8 @@ const nextAddFundsTime = 3 * ledgerUtil.milliseconds.day
 const sufficientBalanceToReconcile = (state) => {
   const balance = Number(ledgerState.getInfoProp(state, 'balance') || 0)
   const unconfirmed = Number(ledgerState.getInfoProp(state, 'unconfirmed') || 0)
-  const bat = ledgerState.getInfoProp(state, 'bat')
-  return bat && (balance + unconfirmed > 0.9 * Number(bat))
+  const budget = parseInt(getSetting(settings.PAYMENTS_CONTRIBUTION_AMOUNT), 10) || 25
+  return balance + unconfirmed >= budget
```

Comment:
The prior code that does `getInfoProp(state, 'bat')`... but I'm not seeing anything which sets this.  I think the field name must have been updated from `bat` to `converted` at some point.  

Why this is important: because ledgerUtil does something similar (in fact, maybe this code can be moved to ledgerUtil?):
https://github.com/brave/browser-laptop/blob/567746835c6c70b1fe2ceecd092953d6d22547df/app/common/lib/ledgerUtil.js#L99-L101

The code in ledgerUtil may have a problem too if nothing is setting this value

Reply:
we need to fix this line as well, we don't have `bat` anymore

Reply:
@NejcZdovc OK great- did you want to do that here also?

Reply:
yeah will fix it in this PR

Reply:
fixed, @bsclifton please re-review